dockerfiles/deb.dockerfile

```diff
@@ -70,9 +70,24 @@ ARG UID=0
 ARG GID=0
 RUN chown -R ${UID}:${GID} /archive /build
 
+# Verify that installing the package succeeds succesfully, and if we're able
+# to run both containerd and runc. This is just a rudimentary check to make
+# sure that package dependencies are installed and that the binaries are not
+# completely defunct.
+FROM distro-image AS verify-packages
+COPY --from=build-packages /build /build
+RUN apt-get update -q \
+ && dpkg --force-depends -i $(find /build -mindepth 3 -type f -name containerd.io_*.deb) || true; \
+    apt-get -y install --no-install-recommends --fix-broken \
+ && apt-get clean \
+ && rm -rf /var/cache/apt /var/lib/apt/lists/*
+RUN containerd --version
```

Comment:
Should we check the exact version? Or maybe move those into a `verify` script and run it here?

Reply:
This step is mostly to detect if the package was not "dead on arrival". 

Having a check to see if the version is matching what we expect would be good to have somewhere, yes. Thinking what the best way would be for that though, because unless we build from a tag, the version is generated;

```
 > [verify-packages 3/5] RUN containerd --version && exit 1:
#26 0.244 containerd containerd.io 20200419.022644~5bfab78a 5bfab78acd08113f49155245fef3aeb8f9f48a83
```